### PR TITLE
Adding avatars as defult

### PR DESCRIPTION
**Pull request by hayesla**
## PR Description
This PR is to use people github avatars rather than a static image

### CODE REVIEW

**Comment by nabobalis** [comment]
I think we will need to update the current documentation in the repo to test it out. 

https://github.com/sunpy/sunpy-sphinx-theme/blob/main/docs/cards.rst

Also remove the stored pngs in the repo as well to reduce the size.

**Comment by Cadair** [comment]
We are keeping the ability to specify a different image right?

**Comment by nabobalis** [comment]
The change as is, suggests no to me

**Comment by nabobalis** [comment]
Should it just not be left alone and then someone can pass in a github url instead? or was the goal to make it less work downstream?

**Comment by hayesla** [comment]
sorry yeah, I'll work on it a little more to try get it working.

I was thinking such that if you have a `img_name` then it will use that, but if theres a `github` handle given and no `img_name` then use the `github` avatar, and then if neither default to the sunpy_icon.svg

**Comment by hayesla** [comment]
> https://github.com/sunpy/sunpy-sphinx-theme/blob/main/docs/cards.rst
> 

also this is amazing lol, I'll test it here

**Comment by hayesla** [comment]
ok this is now working:

https://sunpy--277.org.readthedocs.build/projects/sunpy-sphinx-theme/277/cards.html

If you pass a img_name with a static image it will use that, otherwise if a github handle is given it will pull that avatar or if neither will use default sunpy icon as image.

File: src/sunpy_sphinx_theme/cards.py [hunk 1]
```diff
@@ -22,10 +22,14 @@ def visit_card_node(self, node) -> None:
     key = key.lower().replace(" ", "-")
     title = f"<h4>{title}</h4>" if len(title) > 0 else ""
     col_extra_class = "column-half" if title else ""
+    img_src = node.get("img_name")
+    # if not img_name given, use github avatar if handle given (i.e. check if sunpy_icon defaults)
+    if img_src == "sunpy_icon.svg" and node.get("github") is not None:
+        img_src = f"https://github.com/{node['github']}.png"
```
Comment (nabobalis) [inline src/sunpy_sphinx_theme/cards.py:28]:
```suggestion
    img_src = node.get("img_name")
    # If there is no "img_name" given, we fallback to using the github avatar
    # if a user handle is provided. If so, the image provided is actually the sunpy icon
    if img_src == "sunpy_icon.svg" and node.get("github") is not None:
        img_src = f"https://github.com/{node['github']}.png"
```

**Review by nabobalis — APPROVED** [review]

**Pull request merged**